openedx/core/djangoapps/ace_common/templates/ace_common/edx_ace/common/base_body.html

```diff
@@ -62,7 +62,7 @@
                     <tr>
                         <td width="70">
                             <a href="{% with_link_tracking homepage_url %}"><img
-                                    src="https://media.sailthru.com/595/1k1/8/o/599f355101b3f.png" width="70"
+                                    src="{{ logo_url }}" width="70"
```

Comment:
I’m working on updating ACE templates in another repo. Couldn’t comment on line 49 above: should this be changed to Inter (the new typeface)?

Reply:
@oliviaruizknott Sorry, I am not sure what you mean, can you elaborate a little?

Reply:
My question is about changing the `font-family` to our new typeface Inter (see line **49** above—not the line immediately preceding this comment).

However, it looks like you’re focused only logo changes in this PR—not font changes—so this is probably out of scope. I’ll try to find the person working on font changes for this template. Thanks!